src/index.html

```diff
@@ -5,9 +5,32 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__image" src="images/imac.jpeg"
+      alt="Apple A1419 iMac image">
```

Comment:
nice alt, but it can be little more descriptive [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 13